src/modules/video_player/index.js

```diff
@@ -57,7 +57,32 @@ function handlePlayerClick() {
 }
 
 function handlePlayerScroll(event) {
+  maybeSeek(event);
+  maybeControlVolume(event);
+}
+
+function maybeSeek (event) {
+  if (!settings.get(SettingIds.SCROLL_VIDEO_SEEK)) return;
+  // Non-Alt scrolling controls volume
+  if (!event.altKey) return;
```

Comment:
Should we include the `alt` modifier to toggle between volume control and seeking? We could also not do this, and if a user has both enabled just let it both happen.

Reply:
i would imagine if you are pressing alt it does not affect volume, and if you are not it only affects volume

Reply:
Allright, that's how I made it now 👍